package_scripts/start.ts

```diff
@@ -27,9 +27,7 @@ const hugoServer = (
     renderToDisk: true,
     ...opts
   }
-  return `hugo server ${u.getOptions(
-    allOpts
-  )} --cacheDir="/Users/cchudzicki/dev/ocw-hugo-themes/woof"`
+  return `hugo server ${u.getOptions(allOpts)}`
```

Comment:
I suspect my slack messages while on vacation were confusing. Anyway:
- This change was absolutely an error, thanks for fixing it 👍 
- When i said that the cacheDir stuff was intentional, I meant https://github.com/mitodl/ocw-hugo-themes/blob/main/tests-e2e/global-setup.ts#L43

Reply:
ahh okay
Thanks for letting me know!